file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [22-May-2025 21:29:51 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>acf-field-for-contact-form-7</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[22-May-2025 21:30:00 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>acf-field-for-contact-form-7</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[22-May-2025 21:30:08 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>acf-field-for-contact-form-7</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[22-May-2025 21:30:17 UTC] PHP Fatal error:  Cannot use $this as parameter in /wordpress/wp-content/plugins/acmesocial/unisocrep.php on line 50
[22-May-2025 21:30:24 UTC] PHP Fatal error:  Cannot use $this as parameter in /wordpress/wp-content/plugins/acmesocial/unisocrep.php on line 50
[22-May-2025 21:30:33 UTC] PHP Fatal error:  Cannot use $this as parameter in /wordpress/wp-content/plugins/acmesocial/unisocrep.php on line 50
[22-May-2025 21:31:38 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>admin-commenters-comments-count</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[22-May-2025 21:31:47 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>admin-commenters-comments-count</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[22-May-2025 21:31:55 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>admin-commenters-comments-count</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[22-May-2025 21:32:39 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/adsense-insert/adsenseinsert.php:494
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('adsense-insert/...')
#2 /internal/eval.php(20): activate_plugin('adsense-insert/...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/adsense-insert/adsenseinsert.php on line 494
[22-May-2025 21:32:47 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/adsense-insert/adsenseinsert.php:494
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('adsense-insert/...')
#2 /internal/eval.php(20): activate_plugin('adsense-insert/...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/adsense-insert/adsenseinsert.php on line 494
[22-May-2025 21:32:54 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/adsense-insert/adsenseinsert.php:494
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('adsense-insert/...')
#2 /internal/eval.php(20): activate_plugin('adsense-insert/...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/adsense-insert/adsenseinsert.php on line 494
[22-May-2025 21:34:51 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DROP TABLE wp_jobs;</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_jobs</li>
<li>Executing: DELETE FROM `_wp_sqlite_mysql_information_schema_tables` WHERE `table_schema` = ? AND `table_name` = ? | parameters: database_name_here, wp_jobs</li>
<li>Executing: DELETE FROM `_wp_sqlite_mysql_information_schema_columns` WHERE `table_schema` = ? AND `table_name` = ? | parameters: database_name_here, wp_jobs</li>
<li>Executing: DELETE FROM `_wp_sqlite_mysql_information_schema_statistics` WHERE `table_schema` = ? AND `table_name` = ? | parameters: database_name_here, wp_jobs</li>
<li>Executing: DROP TABLE `wp_jobs` | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_jobs
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('DROP TABLE wp_j...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('DROP TABLE wp_j...')
#3 /wordpress/wp-content/plugins/async-background-worker/async-background-worker.php(60): WP_SQLite_DB->query('DROP TABLE wp_j...')
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('async-backgroun...')
#6 /internal/eval.php(20): activate_plugin('async-backgroun...')
#7 {main}</pre>
 for query DROP TABLE wp_jobs; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/async-background-worker/async-background-worker.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[22-May-2025 21:34:52 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE wp_bg_jobs ADD `created_datetime` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP AFTER `attempts`;</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_bg_jobs</li>
<li>Executing: SELECT COLUMN_NAME FROM `_wp_sqlite_mysql_information_schema_columns` WHERE table_schema = ? AND table_name = ? | parameters: database_name_here, wp_bg_jobs</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_bg_jobs</li>
<li>Executing: 
				SELECT MAX(ordinal_position)
				FROM `_wp_sqlite_mysql_information_schema_columns`
				WHERE table_schema = ?
				AND table_name = ?
			 | parameters: database_name_here, wp_bg_jobs</li>
<li>Executing: INSERT INTO `_wp_sqlite_mysql_information_schema_columns` (`table_schema`, `table_name`, `column_name`, `ordinal_position`, `column_default`, `is_nullable`, `data_type`, `character_maximum_length`, `character_octet_length`, `numeric_precision`, `numeric_scale`, `datetime_precision`, `character_set_name`, `collation_name`, `column_type`, `column_key`, `extra`, `privileges`, `column_comment`, `generation_expression`, `srs_id`) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) | parameters: database_name_here, wp_bg_jobs, created_datetime, 6, CURRENT_TIMESTAMP, NO, timestamp, , , , , 0, , , timestamp, , , select,insert,update,references, , , </li>
<li>Executing: PRAGMA foreign_keys | (no parameters)</li>
<li>Executing: PRAGMA foreign_keys = OFF | (no parameters)</li>
<li>Executing: 
				SELECT *
				FROM `_wp_sqlite_mysql_information_schema_tables`
				WHERE table_type = 'BASE TABLE'
				AND table_schema = ?
				AND table_name = ?
			 | parameters: database_name_here, wp_bg_jobs</li>
<li>Executing: SELECT * FROM `_wp_sqlite_mysql_information_schema_columns` WHERE table_schema = ? AND table_name = ? ORDER BY ordinal_position | parameters: database_name_here, wp_bg_jobs</li>
<li>Executing: 
					SELECT *
					FROM `_wp_sqlite_mysql_information_schema_statistics`
					WHERE table_schema = ?
					AND table_name = ?
					ORDER BY
						INDEX_NAME = 'PRIMARY' DESC,
						NON_UNIQUE = '0' DESC,
						INDEX_TYPE = 'SPATIAL' DESC,
						INDEX_TYPE = 'BTREE' DESC,
						INDEX_TYPE = 'FULLTEXT' DESC,
						ROWID,
						SEQ_IN_INDEX
				 | parameters: database_name_here, wp_bg_jobs</li>
<li>Executing: CREATE TABLE `_wp_sqlite_tmp_wp_bg_jobs_682f987c0dea8` (
  `id` INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
  `queue` TEXT COLLATE NOCASE NOT NULL,
  `payload` TEXT COLLATE NOCASE NOT NULL,
  `attempts` INTEGER NOT NULL,
  `created_datetime` TEXT COLLATE NOCASE NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `created_datetime` TEXT COLLATE NOCASE NOT NULL DEFAULT CURRENT_TIMESTAMP
) STRICT | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 duplicate column name: created_datetime
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('ALTER TABLE wp_...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('ALTER TABLE wp_...')
#3 /wordpress/wp-content/plugins/async-background-worker/async-background-worker.php(72): WP_SQLite_DB->query('ALTER TABLE wp_...')
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('async-backgroun...')
#6 /internal/eval.php(20): activate_plugin('async-backgroun...')
#7 {main}</pre>
 for query ALTER TABLE wp_bg_jobs ADD `created_datetime` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP AFTER `attempts`; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/async-background-worker/async-background-worker.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[22-May-2025 21:35:00 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DROP TABLE wp_jobs;</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DROP TABLE wp_jobs; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jobs.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/async-background-worker/async-background-worker.php(60): WP_SQLite_DB->query('DROP TABLE wp_j...')
#2 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#3 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('async-backgroun...')
#4 /internal/eval.php(20): activate_plugin('async-backgroun...')
#5 {main}</pre>
 for query DROP TABLE wp_jobs; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/async-background-worker/async-background-worker.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[22-May-2025 21:35:01 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE wp_bg_jobs ADD `created_datetime` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP AFTER `attempts`;</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_bg_jobs, `CREATED_DATETIME`, timestamp</li>
<li>Executing: ALTER TABLE wp_bg_jobs ADD `created_datetime` text NOT NULL DEFAULT CURRENT_TIMESTAMP  ; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: created_datetime.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/async-background-worker/async-background-worker.php(72): WP_SQLite_DB->query('ALTER TABLE wp_...')
#2 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#3 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('async-backgroun...')
#4 /internal/eval.php(20): activate_plugin('async-backgroun...')
#5 {main}</pre>
 for query ALTER TABLE wp_bg_jobs ADD `created_datetime` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP AFTER `attempts`; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/async-background-worker/async-background-worker.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[22-May-2025 21:35:09 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DROP TABLE wp_jobs;</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DROP TABLE wp_jobs; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jobs.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/async-background-worker/async-background-worker.php(60): WP_SQLite_DB->query('DROP TABLE wp_j...')
#2 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#3 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('async-backgroun...')
#4 /internal/eval.php(20): activate_plugin('async-backgroun...')
#5 {main}</pre>
 for query DROP TABLE wp_jobs; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/async-background-worker/async-background-worker.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[22-May-2025 21:35:09 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE wp_bg_jobs ADD `created_datetime` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP AFTER `attempts`;</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_bg_jobs, `CREATED_DATETIME`, timestamp</li>
<li>Executing: ALTER TABLE wp_bg_jobs ADD `created_datetime` text NOT NULL DEFAULT CURRENT_TIMESTAMP  ; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: created_datetime.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/async-background-worker/async-background-worker.php(72): WP_SQLite_DB->query('ALTER TABLE wp_...')
#2 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#3 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('async-backgroun...')
#4 /internal/eval.php(20): activate_plugin('async-backgroun...')
#5 {main}</pre>
 for query ALTER TABLE wp_bg_jobs ADD `created_datetime` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP AFTER `attempts`; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/async-background-worker/async-background-worker.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #1 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"akir-hide-text"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/akir-hide-text.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/akir-hide-text.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/akir-hide-text.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/akir-hide-text.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
